Dockerfile

```diff
@@ -1,25 +1,49 @@
+# Base on Python on Debian stable
 FROM python:3-stretch
 
-# Install dependencies for shapely
-RUN apt-get update \
- && apt-get upgrade -y \
- && apt-get install -y libgeos-dev \
- && rm -rf /var/lib/apt/lists/*
-
+EXPOSE 8000
 
-# Uncomment and set with valid connection string for use locally
-#ENV TM_DB=postgresql://user:pass@host/db
+# Install dependencies
+RUN apt-get update \
+  && apt-get upgrade -y \
+  && apt-get install -y libgeos-dev \
+  && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-WORKDIR /src
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
For changing the working directory, I think we need to make a few more modifications (thanks to experience in #1414).

* The directory listed in https://github.com/hotosm/tasking-manager/blob/develop/.ebextensions/aws.config#L12
* Possibly locations where `/app` is mentioned in here? https://github.com/hotosm/tasking-manager/blob/develop/circle.yml

Reply:
* Right, especially for the `aws.config` file! I removed there the `cd /app` command entirely, because the entrypoint should have been defined through the `WORKDIR` anyway.
* I think for the circle it doesn't really matter (it is a separate definition), but if we change to best practices here, I agree we should also do there. I updated the `circle.yml`.